lib/amazon/redshift.rb

```diff
@@ -33,6 +33,18 @@ def reload_from_bucket(bucket, iam_role, encoding: "UTF8", ignore_header: true)
         end
       end
 
+      def reload_from_sql(sql_string)
+        count_before = count
+
+        @connection.transaction do
+          @connection[Sequel.qualify(@schema_sym, @table_name_sym)].truncate
```

Comment:
A thing that I've noticed since we first built this is that `truncate` commits the transaction, so we wouldn't be able to roll this back if inserting failed. Use `.delete`?

Reply:
Fixed.